### PR TITLE
feat: add ens name and avatar support

### PR DESCRIPTION
**Pull request by bigint**
## Description

add ens name and avatar support

Screenshot 2022-01-16 at 8 04 42 PM

## Checklist

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

### CODE REVIEW

**Comment by bigint** [comment]
cc @Atmosfearful

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
This is awesome. Didn't realize resolving was so easy!

Only downside is now it's not clear in the ui how to 'disconnect'. Can you wrap the disconnect button in a `<TextInfoTooltip>` that says 'disconnect'? should be a quick fix.🙏

**Comment by Atmosfearful** [comment]
I'll make sure this makes it to the new site as well!

**Comment by Atmosfearful** [comment]
When you push up a change the failing `cms` build should (hopefully) fix itself

**Comment by bigint** [comment]
@Atmosfearful Can you review now 🙇🏻 

image

File: app/components/views/Home/WalletAction/index.tsx [hunk 1]
```diff
@@ -17,13 +23,17 @@ const WalletAction: FC<WalletProps> = ({
...
     </button>
   ) : (
-    <button
-      type="button"
-      className={styles.disconnectWalletButton}
-      onClick={disconnect}
-    >
-      DISCONNECT WALLET
-    </button>
+    <TextInfoTooltip content="DISCONNECT WALLET" singleton={singleton}>
```
Comment (Atmosfearful) [inline app/components/views/Home/WalletAction/index.tsx:26]: ah no need for the singleton, thats only when you want to animate/sync up multiple tooltips

**Review by Atmosfearful — CHANGES_REQUESTED** [review]
oops meant to request changes. Just remove that singleton and we good to go! Thanks again!

File: app/components/views/Home/WalletAction/index.tsx (hunk 1 above)
Comment (bigint) [reply app/components/views/Home/WalletAction/index.tsx]: @Atmosfearful removed it 👍🏻

**Review by Atmosfearful — APPROVED** [review]

**Comment by bigint** [comment]
@Atmosfearful any ideas when it will go to prod? If available there I'll add the side in ens.domains homepage

**Pull request merged**